### PR TITLE
Installation by multibucket-admin

### PR DESCRIPTION
**Pull request by sunway910**
1 Add: Installation by multibucket-admin
2 update cess latest version in blog
3 fix some content err

please merge after this pr approved: https://github.com/CESSProject/config-gen/pull/9

### CODE REVIEW

File: storage-miner/multi-buckets.md [hunk 1]
```diff
@@ -92,7 +225,7 @@ To run your own RPC node, there are two ways: First is to start through the cess
 
    ⚠️You need to keep the cess-node program running all the time. It is recommended to run cess-node command on a separate window [**screen**](https://linuxize.com/post/how-to-use-linux-screen/) or [**tmux**](https://linuxize.com/post/getting-started-with-tmux/).
```
Comment (jimmychu0807) [inline storage-miner/multi-buckets.md:226]:
```suggestion
   ⚠️You need to keep the cess-node program running all the time. It is recommended to run cess-node command on a separate window using [**screen**](https://linuxize.com/post/how-to-use-linux-screen/) or [**tmux**](https://linuxize.com/post/getting-started-with-tmux/).
```

File: storage-miner/multi-buckets.md [hunk 2]
```diff
@@ -1,16 +1,149 @@
...
+```bash
+  sudo cess-multibucket-admin stop
+```
+
+Stop and remove all containers
+```bash
+  sudo cess-multibucket-admin down
+```
+
+# Run multi-buckets containers step by step
```
Comment (jimmychu0807) [inline storage-miner/multi-buckets.md:132]:
```suggestion
# Run multi-buckets containers manually
```

File: storage-miner/multi-buckets.md [hunk 3]
```diff
@@ -52,7 +185,7 @@ To run your own RPC node, there are two ways: First is to start through the cess
    docker logs chain
    ```
 
-## Run the cess-node program directly
+### Run RPC nodes through source code
```
Comment (jimmychu0807) [inline storage-miner/multi-buckets.md:188]:
```suggestion
### Run RPC node from the source code directly
```

**Review by jimmychu0807 — COMMENTED** [review]
@sunway910 
1. Minor suggestions on wording changes.
2. Also, please resolve the PR conflict with the main branch.

**Comment by tehsunnliu** [comment]
Can you please verify the content @sunway910. 
I'm finding some issues with the doc.
1. In "Run multi-buckets containers by one-click installation" 3rd point it says
"Generate docker-compose.yaml at /opt/cess/multibucket-admin/build/docker-compose.yaml"
But the docker-compose.yaml file content is right at the bottom. Do we need to insert that?
 
2. Over all the doc could be more organized. We can also remove setting up the rpc node section because its already available in https://docs.cess.cloud/core/ref/ops/running-rpc-node.

3. After installing cess-multibucket-admin a config.yaml file is generated in `/opt/cess/multibucket-admin/config.yaml` in which we need to enter our configuration. And right below that it says "Generate each bucket configuration at $diskPath/bucket/config.yaml" but doesn't mention what we need to enter in it? Do we copy the content of multibucket-admin/config.yaml or copy the content from "3. Configure Storage Node Information" step 2?

**Review by tehsunnliu — APPROVED** [review]

**Pull request merged**